### PR TITLE
[TECHPASS-1355] Updated user docs with workaround for signin logs

### PR DESCRIPTION
**Pull request by David-The-Programmer**

### CODE REVIEW

File: docs/account.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# Account
+Account contains your profile and other user settings.
+
+## Profile
+Updating your profile enables TechPass to quickly reach out to you when encountering any difficulties. Rest assured this information is not shared outside of TechPass!
```
Comment (yilunyou) [inline docs/account.md:5]: Change it to "Updating your profile enables TechPass and SGTS products to quickly reach out to you when encountering any difficulties." remove the latter sentence. we will leave the confidential clause to our technical writing team to correct

File: docs/account.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+![profile](assets/images/account/user_profile.png)
+
+!> Updating your TechPass profile does not automatically update your WoG profile. Please approach WoG or your HR directly to get your profile updated.
+
+!> If you are a verified vendor/contractor, please raise a ticket to update your profile [here]().
```
Comment (yilunyou) [inline docs/account.md:11]: u are missing the link in [here]()

Comment (David-The-Programmer) [reply docs/account.md:11]: For this, the link was not there before I made the edit, I checked out the commits, seems to not be there from quite early on

File: docs/account.md (hunk 1 above)
Comment (David-The-Programmer) [reply docs/account.md]: Done

**Pull request merged**